spec.md

```diff
@@ -936,6 +976,8 @@ message NodePublishVolumeResponse {
 
 A Node Plugin MUST implement this RPC call.
 This RPC is a reverse operation of `NodePublishVolume`.
+This RPC MUST remove any mounts setup by the corresponding  `NodePublishVolume`.
```

Comment:
I probably won't mention the word `mount` here because there might be things other than `mount` that this call needs to cleanup. For instance, for the iscsi case, detach will happen after the last `NodeUnpublishVolume`.

I would probably say:
```
This RPC MUST undo the work by the corresponding `NodePublishVolume`.
```